bin/deploy.sh

```diff
@@ -8,7 +8,10 @@ hf -f helmfile.tpl/helmfile-init.yaml template | kubectl apply -f -
 kubectl apply -f charts/prometheus-operator/crds
 
 # helm charts after
-hf apply --skip-deps
+hf apply --skip-deps -l stage!=post
```

Comment:
I see a potential problem with `stage!=post`
E.g.:
```
otomi apply -f helmfile.d/helmfile-30.admin.yaml
```
will deploy every release from `helmfile-30.admin.yaml` 

Do we accept it?

Reply:
That will deploy everything except the ones with `stage=post` which is what we want to yes.